ui/src/routes/devices.$id.tsx

```diff
@@ -841,13 +853,15 @@ export default function KvmIdRoute() {
           />
 
           <div className="relative flex h-full w-full overflow-hidden">
-            <WebRTCVideo />
+            {!isFailsafeMode && <WebRTCVideo />}
```

Comment:
Shouldn't this be only disabled if `isFailsafeMode` is true **and** the reason is video?